addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/handler/KodiHandler.java

```diff
@@ -686,18 +692,38 @@ public void updateDuration(long duration) {
     /**
      * Wrap the given String in a new {@link StringType} or returns {@link UnDefType#UNDEF} if the String is empty.
      */
-    private State createState(String string) {
+    private State createStringState(String string) {
         if (string == null || string.isEmpty()) {
             return UnDefType.UNDEF;
         } else {
             return new StringType(string);
         }
     }
 
+    /**
+     * Wrap the given list of Strings in a new {@link StringType} or returns {@link UnDefType#UNDEF} if the list of
+     * Strings is empty.
+     */
+    private State createStringListState(List<String> list) {
+        if (list == null || list.isEmpty()) {
+            return UnDefType.UNDEF;
+        } else {
+            StringBuilder sb = new StringBuilder();
+            for (String genre : list) {
+                sb.append(genre).append(",");
+            }
+            if (sb.length() > 0) {
+                // drop last comma
+                sb.setLength(sb.length() - 1);
+            }
+            return createStringState(sb.toString());
```

Comment:
This can be simplified with:

```java
return createStringState(list.stream().collect(Collectors.joining(",")));
```
or 

```java
return createStringState(StringUtils.join(list, ","));
```

Reply:
Yes, thanks. I prefer the first one because we do not rely on an external dependency - although we currently use `org.apache.commons.lang.StringUtils` in the Kodi binding.